functions/starter-function/index.js

```diff
@@ -1,13 +1,42 @@
+import middy from '@middy/core';
+import jsonBodyParser from '@middy/http-json-body-parser';
+import validator from '@middy/validator';
+import httpResponseSerializer from '@middy/http-response-serializer';
 import { apiSuccess, apiFailure } from '@utils';
 import { getOrganizations } from '@services/github';
+import { RESPONSE_SERIALIZER } from '@utils/constants';
 
-exports.handler = async (event, _context, callback) => {
-	console.log(JSON.stringify(event));
+export const baseHandler = async (event, _context) => {
 	try {
-		const response = await getOrganizations();
-		return apiSuccess(callback, response);
+		const { organization } = event.body;
+		const response = await getOrganizations(organization);
+		return apiSuccess(response);
 	} catch (error) {
-		console.error(error);
-		return apiFailure(callback, error);
+		return apiFailure(error);
 	}
 };
+
+const inputSchema = {
+	type: 'object',
+	properties: {
+		body: {
+			type: 'object',
+			properties: {
+				organization: { type: 'string' },
+			},
+			required: ['organization'],
+		},
+	},
+};
```

Comment:
I feel the formatting is a little off here too. Maybe GitHub is responsible?

Reply:
Yes, I also think GitHub is the culprit.